packages/blaze/domrange.js

```diff
@@ -332,9 +332,11 @@ DOMRange.prototype.destroy = function (_skipNodes) {
   DOMRange._destroy(this, _skipNodes);
 };
 
-DOMRange.prototype.containsElement = function (elem) {
+DOMRange.prototype.containsElement = function (elem, selector, event) {
+  const templateName = this.view.name?.split('.')[1];
```

Comment:
I would propose to add some fallback string in case we can't resolve the name:

```js
const templateName = this.view?.name
  ? this.view.name.split('.')[1]
  : 'unknown template'
```